src/profile/v100/entity/persona/persona_data/persona_data_identified_entry_types.rs

```diff
@@ -84,6 +84,34 @@ macro_rules! declare_identified_entry {
                 )
             }
         }
+
+        paste! {
+            #[uniffi::export]
+            pub fn [< $struct_name:snake _sample >]() -> $struct_name {
+                $struct_name::sample()
+            }
+
+            #[uniffi::export]
+            pub fn [< $struct_name:snake _sample_other >]() -> $struct_name {
+                $struct_name::sample_other()
+            }
+
+            #[cfg(test)]
+            mod [< uniffi_ $mod_test_name >] {
+                use super::*;
+
+                #[allow(clippy::upper_case_acronyms)]
+                type SUT = $struct_name;
+
+                #[test]
+                fn test_sample_roundtrip() {
```

Comment:
Maybe use same Hash tests as Sargon build information ?

Reply:
The name of this test is a bit confusing, and the hash test is good!

Reply:
Got it